src/modeler/AutoRest.Swagger/Model/ServiceDefinition.cs

```diff
@@ -90,6 +90,7 @@ public ServiceDefinition()
         [CollectionRule(typeof(BodyTopLevelProperties))]
         [CollectionRule(typeof(HttpVerbValidation))]
         [CollectionRule(typeof(DeleteMustHaveEmptyRequestBody))]
+        [CollectionRule(typeof(PropertiesNamesCamelCase))]
```

Comment:
If the previous question is confirmed, and properties are only defined in "definitions", then sure, I can make the tool apply to definitions only.